Assets/Scripts/Managers/PlayerManager.cs

```diff
@@ -5,13 +5,18 @@ namespace Managers
     public class PlayerManager : Singleton<PlayerManager>
     {
         [SerializeField] private GameObject player;
-        
+        [SerializeField] private Inventory inventory;
+
         public GameObject GetPlayer()
         {
             if (player == null)
                 player = GameObject.FindGameObjectWithTag("Player"); //BUG: Replace me with Trigger
             return player;
         }
 
+        public Inventory GetInventory()
+        {
+            return inventory = player.GetComponentInChildren<Inventory>();
```

Comment:
why make it SerializeField if you use GetCompInChildren?

Shouldn't the PlayerManager only have the player anyway? I think you should get the inventory via the player that you get from the playermanager. Correct me, if I'm wrong

Reply:
I'm not sure if PlayerManager should only have the player. If so i will create a getter for inventory in other place.

Reply:
@tobihoffmann @DavidPgl @Kaes3kuch3n @potatoTVnet @thuemmels